src/braket/aws/aws_quantum_task.py

```diff
@@ -381,10 +380,10 @@ def result(
         Consecutive calls to this method return a cached result from the preceding request.
 
         Returns:
-            Union[GateModelQuantumTaskResult, AnnealingQuantumTaskResult, PhotonicModelQuantumTaskResult]: # noqa
+            Union[GateModelQuantumTaskResult, AnnealingQuantumTaskResult, PhotonicModelQuantumTaskResult]:
             The result of the quantum task, if the quantum task completed successfully; returns
             `None` if the quantum task did not complete successfully or the future timed out.
-        """
+        """  # noqa E501
```

Comment:
as above

Reply:
For the E501 issues, I think want to wait and see how that would effect the docs before removing these. In all the cases, it is related to the lengthy type we have in the docstring.